test/development/acceptance-app/hydration-error.test.ts

```diff
@@ -867,7 +874,7 @@ describe('Error overlay for hydration errors in App router', () => {
     await retry(async () => {
       expect(await getRedboxTotalErrorCount(browser)).toBe(
         // One error for "Cannot render a sync or defer <script>"
-        enableOwnerStacks
+        isNewDevOverlay
```

Comment:
For future reference, I'm not a fan of renaming these to their incidental cause. The count is really different because of Owner Stacks not because of new dev overlay. If we'd decouple these in the future, this condition wouldn't make sense.

This caused problems in the past where bugs remained misidentified because they were gated on PPR when they were really caused by experimental React. Correctly stating the flag makes it easier to grep the codebase for impacts of certain flags.